src/test/scala/akka/persistence/jdbc/configuration/AkkaPersistenceConfigTest.scala

```diff
@@ -220,6 +220,11 @@ class AkkaPersistenceConfigTest extends FlatSpec with Matchers {
     cfg.slickConfiguration.jndiDbName shouldBe None
     cfg.slickConfiguration.jndiDbName shouldBe None
 
+    cfg.slickConfiguration.asyncExecutorConfig.numThreads shouldBe 20
```

Comment:
I'm not convinced that this is a useful thing to do. 

The philosophy behind typesafe config is that we should provide default values and let the user override them. This test is only verifying that we do have default values for every property, but that is already provided by resource.conf